tests/test_forward.py

```diff
@@ -0,0 +1,125 @@
+import os
+import sys
+import ipaddress
+
+modules_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
The outmost one is to get the parent dirname. It is needed.

---
In reply to: [99025224](https://github.com/Azure/sonic-snmpagent/pull/4#discussion_r99025224) [](ancestors = 99025224)